packages/react/README.md

```diff
@@ -148,6 +148,21 @@ const useClasses = makeStyles({
 });
 ```
 
+### `shorthands.flex`
```

Comment:
Can you please also add the same to `/apps/website/docs/react/api/shorthands.md`?

Reply:
Done